### DISCUSSION TITLE
Ability to expand large format images

### DISCUSSION THREAD

**Discussion by marionnegp** [RedHat-UX/red-hat-design-system#651] [Ideas]
Dustin Mazon proposed giving users the ability to expand some images, like charts. Should it use a modal or open in a new window/tab in some cases?

![Screen Shot 2022-12-05 at 2 09 49 PM](https://user-images.githubusercontent.com/95588923/205722337-351c0d5f-ab23-42f8-b94d-32e7227ae1c9.png)
- [XD mockup](https://xd.adobe.com/view/2226058f-a9b4-4862-aef3-56cc3058632c-7775/screen/9ba25eff-b4d3-41c9-9457-3a7384aad67b/)
- Customer Portal uses a similar feature to expand their tables in documentation ([example](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/considerations_in_adopting_rhel_9/assembly_hardware-enablement_considerations-in-adopting-rhel-9)).

**Comment by bennypowers** [comment]
What would we use for this if not a modal dialogue? @nikkimk
